R/super_partition.R

```diff
@@ -182,19 +182,25 @@ super_partition <- function(full_data,
       width = 100
     )
   }
-
-  ## first cluster
+  
+  # if no dimension reduction, stop
+  if (length(unique(master_cluster$cluster)) == ncol(full_data)) stop("No dimension reduction occured. Try a lower threshold.")
```

Comment:
Is this really an error? I'm skeptical. We don't error anywhere else when that happens. And should it really stop here, anyway? Why not just then run `partition()` on the original dataset? (Those are actual questions, not rhetorical; you're the expert on `super_partition()`!)

Even if I'm right, I do think it might be worth emitting a message or *maybe* a warning (with a new argument maybe that can suppress the message/warning) that the initial clustering didn't do anything

Reply:
Great point! I've changed this line to use `partition()` instead and included a message to the user if the verbose option is true